src/encoded/static/components/matrix_reference_epigenome.js

```diff
@@ -462,9 +547,24 @@ class MatrixPresentation extends React.Component {
     componentDidMount() {
         this.handleScrollIndicator(this.scrollElement);
 
-        // Display the organism-chooser modal if the query string doesn't specify an organism to
-        // display.
-        this.setState({ organismChooserVisible: this.initialSelectedTab === null });
+        // Highlight body map selections based on url
+        const searchQuery = url.parse(this.props.context['@id']).search;
+        initializeBodyMap(searchQuery);
+        const query = new QueryString(searchQuery);
+        const terms = query.getKeyValues(organField);
+        terms.forEach((term) => {
+            if (CellsList[term] && document.getElementById(term)) {
+                document.getElementById(term).classList.add('active');
+            }
+        });
+
+        this.setState({
+            // Display the organism-chooser modal if the query string doesn't specify an organism to
+            // display.
+            organismChooserVisible: this.initialSelectedTab === null,
```

Comment:
Do you mean null or falsey?

Reply:
I think this is supposed to be null, based on https://github.com/ENCODE-DCC/encoded/blob/4534124558358e33fec5ed399141146ff3ffcc4d/src/encoded/static/components/matrix_reference_epigenome.js#L622. I think I didn't change this line, I just added another state change to the setState.